chapters/hub/hub-api.md

```diff
@@ -0,0 +1,66 @@
+# Jina Hub API Reference
+
+Jina Hub API allows users to :`login`: to Hub, :`list`:, :`build`:, :`push`: and create :`new`: Hub images with configurable parameters. This guide covers installing and setting up the API for a seamless experience with the Hub.
+
+
+## Install:
+Run `pip install "jina[hub]"`
+
+## `jina hub list`:
```

Comment:
Can you use **J**ina **H**ub list: ? Same for all the others?

Reply:
`jina hub list` is a CLI command (since it's wrapped in backticks) so it should be in the format as currently written.

Whether the command itself should be the section heading is another matter. I have no strong opinion on it. However, if we do change the section heading we need to be more explicit about the command in the text below (i.e. `jina hub list`, not just `hub list`)

Reply:
Yes, the heading looks good with the command itself